docs/docs-beta/docs/guides/automation/asset-sensors.md

```diff
@@ -1,10 +1,80 @@
 ---
-title: Asset Sensors
-sidebar_position: 50
+title: Scheduling pipelines across projects and jobs with Asset Sensors
```

Comment:
This page covers more than one type of asset sensor (`@asset_sensor` and `@multi_asset_sensor`). We should include some kind of 'rollup' of the different types of asset sensors and when to use each one before jumping into the individual sections.

Reply:
Ive been thinking through this and have struggled to find the right way to bring it up in this page. 

I think it would be better in a concept section. If we think about the tasks people want to do, then either the @asset_sensor or @multi_asset_sensor is the right tool for that job. 

But if people want to know about the different types of asset sensors, then that goes into a concept/explainer page. 

I could add maybe a sentence under the `Monitor multiple assets` section. 

What do you think of this:

```
The previous examples showed how to use a single asset sensor to monitor a single asset and trigger a job when it is materialized. In this example, we'll show how to use a multi-asset sensor to monitor multiple assets and trigger a job when any of the monitored assets are materialized.
```

Reply:
I dig that.